src/main/java/org/assertj/core/api/AbstractObjectArrayAssert.java

```diff
@@ -88,7 +88,6 @@
  * @author Mateusz Haligowski
  * @author Lovro Pandzic
  */
-@SuppressWarnings("deprecation")
```

Comment:
this one is needed because of the now deprecated but imported `FieldByFieldComparator` and `IgnoringFieldsComparator` (line 62-63)

Reply:
Interesting. When I compile the the class with Java I only get warnings on lines: the imports (line 62, 63 and 68) and also on line 426 for the `containsOnlyElementsOf`.

When I bring back the `SuppressWarnings` the warning on line 426 is gone, but the ones for the imports are still there.

Reply:
if we could apply the SuppressWarnings at the import level that would be great but since it's not possible let's bring back this SuppressWarnings, it's not fine grained but I don't want to see deprecation warnings in my eclipse IDE ;-)